### PR TITLE
*: update all registry deps

### PR DESCRIPTION
**Pull request by brson**
Signed-off-by: Brian Anderson <andersrb@gmail.com>

###  What have you changed?

Upgrade all non-git dependencies to their latest, updating lockfile.

###  What is the type of the changes?

Pick one of the following and delete the others:

- Engineering (engineering change which doesn't change any feature or fix any issue)

###  How is the PR tested?

cargo test --all --features=jemalloc,sse,portable

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

no

###  Does this PR affect `tidb-ansible`?

no

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2107,86 +2216,84 @@ dependencies = [
...
 ]
 
-[[package]]
-name = "rle-decode-fast"
-version = "1.0.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "cabe4fa914dec5870285fa7f71f602645da47c486e68486d2b4ceb4a343e90ac"
-
 [[package]]
 name = "rocksdb"
```
Comment (5kbpers) [inline Cargo.lock]: Is it OK to update `rocksdb`? PTAL @yiwu-arbug @Connor1996

Comment (brson) [reply Cargo.lock]: oh good catch i didn't mean to update any git repos. I'll fix it.

**Comment by brson** [comment]
ok @5kbpers I restored the previous git revs for rocks and libfuzzer. ptal and @sticnarf

**Pull request merged**